package.json

```diff
@@ -17,6 +17,7 @@
   "dependencies": {
     "async": "=2.4.1",
     "bignumber.js": "=4.0.2",
+    "bluebird": "=3.5.0",
```

Comment:
Ok no problem. Yes keeping to the same library application-wide would be ideal.

Should we switch pg-promise too? https://github.com/vitaly-t/pg-promise#promiselib